assets/src/edit-story/elements/shape/index.js

```diff
@@ -32,7 +32,7 @@ export const defaultAttributes = {
   focalY: 50,
 };
 
-export const hasEditMode = true;
+export const hasEditMode = ({ backgroundImage }) => Boolean(backgroundImage);
```

Comment:
This meant to work a bit differently. The mask applies to any element, in theory. So, we'd never have a case when a shape has a backgroundImage - it'd simply be the image element, or a video element, etc.